pom.xml

```diff
@@ -126,6 +128,11 @@
                 <artifactId>fest-assert</artifactId>
                 <version>${version.fest-assert}</version>
             </dependency>
+            <dependency>
+                <groupId>org.assertj</groupId>
+                <artifactId>assertj-core</artifactId>
```

Comment:
> The problem is that I don't know how many tests we should refactor. 

This is searchable :)

> I think this should be done in another PR.

I believe it should belong in this one. It's really easy.

http://joel-costigliola.github.io/assertj/assertj-core-migrating-from-fest.html